libs/viewer/src/TweakableMaterial.cpp

```diff
@@ -309,7 +330,6 @@ void TweakableMaterial::drawUI(const std::string& header) {
     case MaterialType::Subsurface: {
         if (ImGui::CollapsingHeader("Subsurface settings")) {
             mSubsurfaceColor.addWidget("subsurface color");
-            mSheenColor.addWidget("sheen color");
```

Comment:
just curious: will `sheenColor` be retired (deleted) from Subsurface.mat? If so, when? In this PR?

Reply:
It has already been retired in this PR.

Reply:
Sorry, I missed the fact that `sheenColor` was not a parameter in Subsurface.mat in the first place and you only removed the UI widget now. 🙄